includes/wc-term-functions.php

```diff
@@ -395,11 +395,21 @@ function wc_reorder_terms( $the_term, $next_id, $taxonomy, $index = 0, $terms =
 		// set order
 		$index++;
 		$index = wc_set_term_order( $term->term_id, $index, $taxonomy );
+		
+		/**
+		 * After a term has had it's order set.
+		*/
+		do_action( 'woocommerce_set_term_order', $term, $index, $taxonomy );
```

Comment:
Yeah I named it like that because I'm not sure how you normally decide upon action names. I chose something close to what I've seen you using elsewhere.